src/frontend/components/CallAssignment.js

```diff
@@ -142,20 +143,32 @@ class CallAssignment extends React.Component {
   }
 
   renderIntervieweeInfo() {
-    let interviewee = this.props.currentUser.intervieweeForCallAssignment;
-    let name = this.intervieweeName();
-    let number = this.formatPhoneNumber(interviewee.phone)
+    let interviewee = this.props.currentUser.intervieweeForCallAssignment
+    let name = this.intervieweeName()
+    let formattedNumber = this.formatPhoneNumber(interviewee.phone)
+    let plivoStatusText = this.state.plivoStatusText;
+    let plivoCallInProgress = this.state.plivoCallInProgress;
 
     let sideBar = (
-      <div style={{
-        ...BernieText.default,
-        color: BernieColors.blue,
-        fontSize: '1.5em',
-        fontWeight: 600
-      }}>
-        {name}
-        <br />
-        {number}
+      <div>
+        <div style={{
+          ...BernieText.default,
+          color: BernieColors.blue,
+          fontSize: '1.5em',
+          fontWeight: 600
+        }}>
+          {name}
+          <br />
+          <a href={`tel:+1${interviewee.phone}`}
+             style={{color: BernieColors.blue,
+                     textDecoration: 'none'}}>
+            {formattedNumber}
+          </a>
+          <PlivoDialer number={`1${interviewee.phone}`}
+            endpointUsername='bernie151217012337'
```

Comment:
No whitelisting or other reasonable security practices on Plivo's side from what I gather :cold_sweat: 

Yes, you should be using Direct Dial. At least, that's what I tested with. The docs don't indicate that going to production while relying on their Direct Dial app is unacceptable, although they do provide instructions on how we would replicate that functionality if we wanted to host our own dialer app.